sekoia_automation/connector/__init__.py

```diff
@@ -263,20 +311,29 @@ def _chunk_events(self, events: Sequence) -> Generator[list[Any], None, None]:
 
         # iter over the events
         for event in events:
-            if len(event) > EVENT_BYTES_MAX_SIZE:
+            result_event = str(event)
+
+            if isinstance(event, BaseModel):
+                result_event = orjson.dumps(event.dict()).decode("utf-8")
```

Comment:
Isn't Pydantic model [`model_dump_json`](https://docs.pydantic.dev/latest/api/base_model/#pydantic.BaseModel.model_dump_json) more suitable?
```suggestion
                result_event = event.model_dump_json()
```

Reply:
Basically we are using version `pydantic = "^1.10"` and in lock we have `version = "1.10.18"`
If I'm correct this is available only from version 2
In version that we are using we have only model_dump that produce dict...
https://docs.pydantic.dev/latest/concepts/serialization/#modelmodel_dump